modules/superset/tests/basic/main.tf

```diff
@@ -0,0 +1,19 @@
+module "test" {
+  source = "../../"
+
+  name               = "superset"
+  namespace          = "superset"
+  cluster_name       = "buycycle-cluster"
+  supeset_secret_key = "sdgiadgiagdiagdi"
```

Comment:
@aghamyan44 do we need this? shouldn't basic be minimal? or maybe we should call this minimal?